src/Rules/StarterWeb/AI/IndividualAuth/project.json

```diff
@@ -61,6 +58,13 @@
         "portable-net45+win8"
       ]
     },
+    "Microsoft.Extensions.CodeGeneration.Tools": {
+      "version": "__AspNetCorePlatformVersion__",
+      "imports": [
+        "portable-net45+win8+dnxcore50",
```

Comment:
Yes, net45+win8+dnxcore50 is required to get the proper version of json.net, just specifying net45+win8 nets you a json.net that has a fx dependency on System.Runtime 2.0.5.0

net45+win8 is required for Ix.Async and Remotion